src/Razor/src/Microsoft.AspNetCore.Razor.Language/Components/ComponentMetadata.cs

```diff
@@ -17,16 +17,6 @@ internal static class ComponentMetadata
 
         public static readonly string ImportsFileName = "_Imports.razor";
 
-        public static bool IsComponentTagHelperKind(string tagHelperKind)
-        {
-            return tagHelperKind == Component.TagHelperKind ||
-                tagHelperKind == ChildContent.TagHelperKind ||
-                tagHelperKind == EventHandler.TagHelperKind ||
-                tagHelperKind == Bind.TagHelperKind ||
-                tagHelperKind == Key.TagHelperKind ||
-                tagHelperKind == Ref.TagHelperKind;
-        }
```

Comment:
This was the cause of about an hour of debugging on my part before I understand why **just** the splat tag helper wasn't showing up. @ajaybhargavb - remind me to tell you a story about this.